### PR TITLE
Add quick stats endpoint

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -13669,23 +13669,25 @@ Managing calendar reminders
   :statuscode 500: Internal rotki error.
 
 
-.. http:delete:: /api/(version)/calendar/reminders
```
Comment (LefterisJP) [inline docs/api.rst:13672]: Are you deleting the calendar reminder api docs entry? Seems so from the github diff

File: docs/api.rst [hunk 2]
```diff
@@ -13695,12 +13697,44 @@ Managing calendar reminders
...
+              "POLYGON_POS": 186,
+              "SCROLL": 17
+          }
+        },
+        "status_code": 200
       }
 
-  :resjson bool result: Boolean denoting success or failure.
+  :resjson string eth_on_gas: Amount of ETH spent on gas.
+  :resjson object gnosis_max_payments_by_currency: Object that maps currency symbol to the max amount paid in a single payments using that currency.
```
Comment (LefterisJP) [inline docs/api.rst:13734]: you have this two times no? `gnosis_max_payments_by_currency` ?

Is this only for gnosis pay?

File: rotkehlchen/db/history_events.py [hunk 3]
```diff
@@ -802,3 +803,50 @@ def edit_event_extra_data(
             (json.dumps(extra_data), event.identifier),
         )
         event.extra_data = extra_data
+
+    def query_wrap_stats(self, from_ts: Timestamp, to_ts: Timestamp) -> dict[str, Any]:
+        """Query simple statistics about the user to show them as a wrap for the year"""
```
Comment (LefterisJP) [inline rotkehlchen/db/history_events.py:808]: Here and in everything else can you specify in the docstrings this is a temporary endpoint?

File: rotkehlchen/db/history_events.py [hunk 4]
```diff
@@ -802,3 +803,50 @@ def edit_event_extra_data(
...
+        from_ts_ms, to_ts_ms = ts_sec_to_ms(from_ts), ts_sec_to_ms(to_ts)
+        with self.db.conn.read_ctx() as cursor:
+            cursor.execute(
```
Comment (LefterisJP) [inline rotkehlchen/db/history_events.py:811]: for the gas, also in the api make it a sum and a per address one.

File: rotkehlchen/db/history_events.py [hunk 5]
```diff
@@ -802,3 +803,50 @@ def edit_event_extra_data(
...
+                (from_ts, to_ts),
+            )
+            transactions_per_chain = {ChainID.deserialize_from_db(row[0]).name: row[1] for row in cursor}  # noqa: E501
+            cursor.execute(
+                'SELECT location, COUNT(*) from trades '
+                'WHERE timestamp >= ? AND timestamp <= ? GROUP BY location',
+                (from_ts, to_ts),
+            )
+            trades_by_exchange = {str(Location.deserialize_from_db(row[0])): row[1] for row in cursor}  # noqa: E501
+            cursor.execute(
```
Comment (LefterisJP) [inline rotkehlchen/db/history_events.py:831]: this is only for users with premium + gnosis pay right? So perhaps if there is nothing return empty? How should this work?

File: docs/api.rst [hunk 6]
```diff
@@ -13695,12 +13697,44 @@ Managing calendar reminders
...
+        "message": "",
+        "result": {
+          "eth_on_gas": "0.21760082633497774",
+          "gnosis_max_payments_by_currency": {
+              "AED": "36.0",
+              "EUR": "120.17",
+              "THB": "13615.25",
+              "USD": "85.0"
+          },
+          "top_days_by_number_of_transactions": [
```
Comment (LefterisJP) [inline docs/api.rst:13709]: what is this? It's not explained in the docs well and not sure I understand.

File: rotkehlchen/db/history_events.py [hunk 7]
```diff
@@ -802,3 +803,50 @@ def edit_event_extra_data(
...
+                'SELECT timestamp, COUNT(*) as tx_count FROM evm_transactions '
+                'WHERE timestamp >= ? AND timestamp <= ? GROUP BY timestamp ORDER BY '
+                'tx_count DESC LIMIT 10',
+                (from_ts, to_ts),
+            )
+            top_days_by_number_of_transactions = [{'timestamp': row[0], 'amount': str(row[1])} for row in cursor]  # noqa: E501
+
+        return {
+            'eth_on_gas': eth_on_gas,
```
Comment (LefterisJP) [inline rotkehlchen/db/history_events.py:847]: I would add a number of transactions per protocol.

File: docs/api.rst (hunk 6 above)
Comment (yabirgb) [reply docs/api.rst:13709]: it is like the heatmap on github. Returns the 10 days when you transacted the most

File: rotkehlchen/db/history_events.py (hunk 5 above)
Comment (yabirgb) [reply rotkehlchen/db/history_events.py:831]: if there is not information it will be empty

File: docs/api.rst (hunk 2 above)
Comment (yabirgb) [reply docs/api.rst:13734]: yes appears twice. There is a entry only for gnosis pay

File: docs/api.rst [hunk 8]
```diff
@@ -13704,3 +13704,85 @@ Managing calendar reminders
...
+      }
+
+  :resjson string eth_on_gas: Amount of ETH spent on gas.
+  :resjson object eth_on_gas_per_address: Amount of ETH spent on gas per address.
+  :resjson object gnosis_max_payments_by_currency: Object that maps currency symbol to the max amount paid in a single payments using that currency.
+  :resjson list top_days_by_number_of_transactions: list with the top 10 days with more EVM transactions. Sorted in descending order.
+  :resjson list transactions_per_protocol: Sorted list of per protocol interactions by number of transactions.
+  :resjson object trades_by_exchange: number of trades executed on every tracked exchange.
+  :resjson object transactions_per_chain: number of transactions that rotki tracks by chain for the user.
+  :statuscode 200: Entry correctly deleted.
```
Comment (LefterisJP) [inline docs/api.rst:13786]: copy pasta?

File: rotkehlchen/tests/api/test_statistics.py [hunk 9]
```diff
@@ -431,3 +457,194 @@ def mock_premium_get(url: str, *_args: Any, **_kwargs: Any) -> MockResponse:
...
         )
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0x01471dB828Cfb96Dcf215c57a7a6493702031EC1']])
+@pytest.mark.parametrize('base_accounts', [['0x01471dB828Cfb96Dcf215c57a7a6493702031EC1']])
+def test_query_wrap(
+        rotkehlchen_api_server: 'APIServer',
+        ethereum_accounts: list[ChecksumEvmAddress],
+) -> None:
+    """Test that information returned by recap endpoint is correct.
```
Comment (LefterisJP) [inline rotkehlchen/tests/api/test_statistics.py:468]:
```suggestion
    """Test that information returned by the yearly wrap endpoint is correct.
```

**Review by LefterisJP — APPROVED** [review]
Please change wrap -> yearly wrap or year's wrap everywhere.

**Pull request merged**